scripts/figures/difference/diff_map_cba.R

```diff
@@ -53,18 +53,18 @@ ggplot() +
   geom_basemap() +
   coord_sf_mal() +
   annotate_map(
-    title = "Henkilöauton matka-aikamuutos asukasta kohden",
+    title = "Henkilöauton matka-aikamuutos kiertomatkaa kohden",
     subtitle = "2040 Vertailupohja \U2192 2040 1. luonnos"
   ) +
   theme_mal_map()
 
-ggsave_map(here::here("figures", sprintf("map_cba_car-time_%s.png", scenario_attributes[["scenario"]])))
+ggsave_map(here::here("figures", sprintf("map_cba_car-time_tour_%s.png", scenario_attributes[["scenario"]])))
 
 
 # Plot --------------------------------------------------------------------
 
 ggplot() +
-  geom_sf(mapping = aes(fill = cba_transit_time_per_person + cba_car_time_per_person),
+  geom_sf(mapping = aes(fill = cba_transit_time_per_tour + cba_car_time_per_tour),
```

Comment:
@zptro I said wrong, here is how I calculated "Joukkoliikenteen ja henkilöauton yhteenlaskettu matka-aikamuutos kiertomatkaa kohden". Should it actually be "sum of travel time changes" / "sum of car and transit tours"?

Reply:
I will fix this!

Reply:
Fixed!